server/app/controllers/NewTask.scala

```diff
@@ -39,13 +43,16 @@ class NewTask @Inject()(executor: DynamicSuiteExecutor, dao: Dao, val messagesAp
       f => {
         val futureResponse = for {
           _ <- Future(StringBuilderRunner(executor(f.referenceSolution, f.suite))).check
-          db <- dao.addTask(Task(scalaClass, f.taskDescription, f.solutionTemplate, f.referenceSolution, f.suite))
+          db <- dao.addTask(
+            Task(new Timestamp(System.currentTimeMillis),
```

Comment:
First of all currentTime is ignored, so we can do it Optional. The second one - do we really need java.sql.Timestamp? Cassandra do not preserve timezone, System.currentTimeMillis could be enough in my opinion